mayastor-test/e2e/node_disconnect/node_disconnect_iscsi/node_disconnect_iscsi_test.go

```diff
@@ -0,0 +1,128 @@
+package node_disconnect_iscsi_test
+
+import (
+	"e2e-basic/common"
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var defTimeoutSecs = "90s"
+
+var g_nodeToKill = ""
+var g_nexusNode = ""
+var g_uuid = ""
```

Comment:
FYI - I believe you can also declare these in the following way:
var (
   defTimeoutSecs = "90s"
   g_nodeToKill = ""
   g_nexusNode = ""
   g_uuid = ""
)